apps/cookbook/src/app/showcase/item-showcase/item-showcase.component.html

```diff
@@ -270,6 +270,29 @@ <h2 #disclosure>Disclosure</h2>
   </em>
 </p>
 
+<h4>Rotation animation</h4>
+
+<p>
+  It is possible to trigger an animation that rotates the disclosure icon by passing a boolean flag
+  to
+  <code>[rotateIcon]</code>
+  .
+</p>
+
+<cookbook-example-viewer [html]="disclosureAnimationExample.template">
+  <cookbook-item-example-disclosure-animation
+    #disclosureAnimationExample
+  ></cookbook-item-example-disclosure-animation>
+</cookbook-example-viewer>
+
+<p class="comment small">
+  <em>
+    <strong>Please note</strong>
+    , the animation of the disclosure icon is only available for:
```

Comment:
Consider readability:
```suggestion
    <strong>Please note:</strong>
    the animation of the disclosure icon is only available for:
```